src/assets/mockups/indicatorsDashData.jsx

```diff
@@ -2,91 +2,102 @@
 const arrayData = [
```

Comment:
You should delete this file since it's not being used because you are calling the mockup from the backend